styles/twitch/catppuccin.user.css

```
@@ -711,6 +711,382 @@
   }
 }
 
+@-moz-document url("https://dev.twitch.tv/"), url-prefix("https://dev.twitch.tv/docs"), url-prefix("https://dev.twitch.tv/code"), url-prefix("https://dev.twitch.tv/products"), url-prefix("https://dev.twitch.tv/showcase"), url-prefix("https://dev.twitch.tv/support"), url-prefix("https://discuss.dev.twitch.com/embed/topics")
```

Is there a reason we need to do all these prefixes?
```suggestion
@-moz-document domain("dev.twitch.tv")
```